cmd/mattermost-mattermod/main.go

```diff
@@ -27,8 +28,14 @@ func main() {
 	}
 	server.SetupLogging(config)
 
+	// Metrics system
+	metricsProvider := metrics.NewPrometheusProvider()
+	metricsServer := metrics.NewServer("8067", metricsProvider.Handler(), true)
+	metricsServer.Start()
+	defer metricsServer.Stop()
```

Comment:
Should this started and terminated by `server`?

Reply:
I want to decouple the metrics system from the server system taking advantage that we are using it for the cron tasks too. So I think is better to start/stop it outside the server cycle